### DISCUSSION TITLE
Hackathon projects presentation

### DISCUSSION THREAD

**Discussion by filippo82** [softwareunderground/transform-2021-hackathon#25] [General]
Hello hackers!

**Who wants to present the results of your hackathon project in a 5 minutes video or live presentation on Friday?**

Please comment below using this template:

```
{Slack channel name}
{Slack username of the presenter}
{Live or video}
{Time zone}
```

**ONLY ONE presentation per project** 😇 

Thank you!
-Filippo

`t21-hack-*` channel list:
- [x] t21-hack-css
- [x] t21-hack-footprint
- [x] t21-hack-gemgis
- [x] t21-hack-ocean-acoustics <- NEEDS TO BE THE FIRST!
- [ ] t21-hack-segysak <- Tony is the only one working on it and cannot do it live
- [x] t21-hack-stats
- [x] t21-hack-striplog
- [x] t21-hack-subsurface
- [x] t21-hack-wellcor

** Not presenting **
- [ ] ~t21-hack-connections~
- [ ] ~t21-hack-qgis_tools~
- [ ] ~t21-hack-lasio~

**Comment by prisae** [comment]
- #t21-hack-subsurface
- Leguark & prisae
- Live
- UTC +2

**Comment by mlbelobraydi** [comment]
- #t21-hack-wellcorr "SwellCorr"
- matthew belobraydic or evan
- Live
- UTC -5

**Comment by mtb-za** [comment]
* #t21-hack-striplog
* mtb-za (Martin Bentley)
* Live
* UTC +2

**Comment by JustinGOSSES** [comment]
* #t21-hack-stats
* Justin Gosses
* Live
* UTC -5

I've also signed up for a lightning talk already and will use that 10 min  to mention the larger goal behind stats & connections project ideas

**Comment by AlexanderJuestel** [comment]
- #t21-hack-gemgis
- AlexanderJuestel
- Live
- UTC +2

**Comment by filippo82** [comment]
* #t21-hack-ccs
* Nathan Jones
* Live
* UTC -7

**Comment by filippo82** [comment]
* #t21-hack-footprint
* Elwyn Galloway
* Live
* UTC -6

**Comment by Artash-N** [comment]
- #t21-hack-ocean-accoustics
- Artash-N
- Live
- UTC -4
